package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@128technology/yinz",
-  "version": "4.2.12",
+  "version": "4.2.13",
```

Comment:
Can you bump it in `package-lock.json` for consistency too, even though it doesn't have functional impact in there